src/docs/index.rst

```diff
@@ -55,13 +58,16 @@ for Axom software components:
 Component User Guides
 ----------------------
 
-  *  Slic (Simple Logging Interface Code for integrated applications)
-  *  `Lumberjack (Scalable parallel message logging and filtering) <../../lumberjack_docs/html/index.html>`_
-  *  `Sidre (Simulation data repository) <../../sidre_docs/html/index.html>`_
-  *  `Slam (Set-theoretic lightweight API for meshes) <../../slam_docs/html/index.html>`_
-  *  `Quest (Querying on surface tool) <../../quest_docs/html/index.html>`_
-  *  `Mint (Mesh data model) <../../mint_docs/html/index.html>`_
-  *  `Primal (Computational geometry primitives) <../../primal_docs/html/index.html>`_
+.. toctree::
+   :maxdepth: 1
+
+   Slic (Simple Logging Interface Code) <../axom/slic/docs/sphinx/index>
+   Lumberjack (Scalable parallel message logging and filtering) <../axom/lumberjack/docs/sphinx/index>
+   Sidre (Simulation data repository) <../axom/sidre/docs/sphinx/index>
+   Slam (Set-theoretic lightweight API for meshes) <../axom/slam/docs/sphinx/index>
+   Slam (Set-theoretic lightweight API for meshes) <../axom/quest/docs/sphinx/index>
```

Comment:
Typo: the second `Slam` should be `Quest`

Reply:
Thanks!